en/scale-a-tidb-cluster.md

```diff
@@ -12,13 +12,47 @@ This document introduces how to horizontally and vertically scale a TiDB cluster
 
 Horizontally scaling TiDB means that you scale TiDB out or in by adding or remove nodes in your pool of resources. When you scale a TiDB cluster, PD, TiKV, and TiDB are scaled out or in sequentially according to the values of their replicas. Scaling out operations add nodes based on the node ID in ascending order, while scaling in operations remove nodes based on the node ID in descending order.
 
-Currently, the TiDB cluster supports management by Helm or by TidbCluster Custom Resource (CR). You can choose the scaling method based on the management method of your TiDB cluster.
+Currently, the TiDB cluster supports management by TidbCluster Custom Resource (CR).
 
-### Horizontal scaling operations (CR)
+### Scale PD, TiDB, and TiKV
 
-#### Scale PD, TiDB, and TiKV
+Modify `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` in the `TidbCluster` object of the cluster to a desired value using kubectl. You can modify the values in the local file or using online command.
 
-Modify `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` in the `TidbCluster` object of the cluster to a desired value using kubectl.
+- If a yaml file that describes the TiDB cluster exists in your local machine, modify `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` in the local file to your desired values. Then deploy the yaml file to the cluster by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl apply -f ${target_file}.yaml -n ${namespace}
+    ```
+
+- You can also online modify the `TidbCluster` definition in the Kubernetes cluster by running the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl edit tidbcluster ${cluster_name} -n ${namespace}
+    ```
+
+After modifying the values above, check if the TiDB cluster in Kubernetes has updated to your desired definition:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl get tidbcluster ${cluster_name} -n ${namespace} -oyaml
+```
+
+In the `TidbCluster` file output by the command above, if the values of `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` are consistent with the values you have modified, check whether the number of `TidbCluster` Pods has increased or decreased by running the following command:
```

Comment:
```suggestion
In the `TidbCluster` file output by the command above, if the values of `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` are consistent with the values you have modified, check whether the number of `TidbCluster` Pods have increased or decreased by running the following command:
```

Reply:
I think the subject `number` is singular, so we should use `has` instead of `have`.